lib/applicant_factory.rb

```diff
@@ -23,7 +23,7 @@ def make_a_person(applicant)
         work_phone: Faker::PhoneNumber.phone_number,
         home_phone: Faker::PhoneNumber.phone_number,
         cell_phone: Faker::PhoneNumber.phone_number,
-        citizenship: Faker::Address.country,
+        citizenship: ["US Citizen", "Non-Citizen with eligible immigration status", "Other"].sample,
```

Comment:
Good catch.  I updated Person#us_citizen? and that broke a test because it was looking for "United States" instead of "US Citizen". Let me know if there is anything else.